src/main/java/pl/widulinski/scrapp/webDriver/BrowserPicker.java

```diff
@@ -0,0 +1,27 @@
+package pl.widulinski.scrapp.webDriver;
+
+
+import lombok.Data;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.springframework.stereotype.Component;
+
+@Component
+@Data
```

Comment:
jesli gdzies robisz `getDriver` to dodaj po prostu `@Getter` a nie całe `@Data`

Reply:
klasa wywalona